docs/kubernetes/windows.md

```diff
@@ -1,446 +1,358 @@
-        "count": 2,
-        "vmSize": "Standard_D2_v3",
-        "availabilityProfile": "AvailabilitySet",
-        "osType": "Windows",
-        "osDiskSizeGB": 127
-     }
-```
+> If you want other versions, check out the [official instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). For more help, check out the Azure CLI [getting started](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli?view=azure-cli-latest) page.
 
-#### Choosing the Windows Server version
+#### ACS-Engine
 
-If you want to deploy a specific Windows Server version, you can find available versions with `az vm image list --publisher MicrosoftWindowsServer --all -o table`
+Windows support is evolving rapidly, so be sure to use the latest ACS-Engine  version (v0.20 or later).
 
-```
-$ az vm image list --publisher MicrosoftWindowsServer --all -o table                                                                                        
+1. Browse to the ACS-Engine [releases page](https://github.com/Azure/acs-engine/releases) on GitHub.
 
-Offer                    Publisher                      Sku                                             Urn                                                                                                            Version
------------------------  -----------------------------  ----------------------------------------------  -------------------------------------------------------------------------------------------------------------  -----------------
-...
-WindowsServerSemiAnnual  MicrosoftWindowsServer         Datacenter-Core-1709-with-Containers-smalldisk  MicrosoftWindowsServer:WindowsServerSemiAnnual:Datacenter-Core-1709-with-Containers-smalldisk:1709.0.20180412  1709.0.20180412
-WindowsServerSemiAnnual  MicrosoftWindowsServer         Datacenter-Core-1803-with-Containers-smalldisk  MicrosoftWindowsServer:WindowsServerSemiAnnual:Datacenter-Core-1803-with-Containers-smalldisk:1803.0.20180504  1803.0.20180504
-```
+2. Find the latest version, and download the file ending in `-windows-amd64.zip`.
 
-You can use the Offer, Publisher and Sku to pick a specific version by adding `windowsOffer`, `windowsPublisher`, `windowsSku` and (optionally) `widndowsVersion` to the `windowsProfile` section. In this example, the latest Windows Server version 1803 image would be deployed.
+3. Extract the `acs-engine...-windows-amd64.zip` file to a working folder such as `c:\tools`
 
-```
-"windowsProfile": {
-            "adminUsername": "azureuser",
-            "adminPassword": "...",
-            "windowsPublisher": "MicrosoftWindowsServer",
-            "windowsOffer": "WindowsServerSemiAnnual",
-            "windowsSku": "Datacenter-Core-1803-with-Containers-smalldisk"
-     },
+4. Check that it runs with `.\acs-engine.exe version`
+
+```none
+PS C:\Users\patrick\acs-engine> .\acs-engine.exe version
+Version: v0.20.6
+GitCommit: 293adfda
+GitTreeState: clean

```

Comment:
Provide second example for macOS / Linux:
 /usr/local/bin/acs-engine version                                                                                                                 Version: v0.20.6
GitCommit: 293adfda
GitTreeState: clean